sql/parser/ast.go

```diff
@@ -205,22 +308,33 @@ const (
 	astDropTable       = "DROP TABLE"
 	astDropView        = "DROP VIEW"
 	astRenameTable     = "RENAME TABLE"
+	astShowDatabases   = "SHOW DATABASES"
 	astShowTables      = "SHOW TABLES"
 	astShowIndex       = "SHOW INDEX FROM"
 	astShowColumns     = "SHOW COLUMNS FROM"
 	astShowFullColumns = "SHOW FULL COLUMNS FROM"
 	astTruncateTable   = "TRUNCATE TABLE"
+	astIfNotExists     = " IF NOT EXISTS"
+	astUnique          = "UNIQUE"
+	astPrimaryKey      = "PRIMARY KEY"
+	astKey             = "KEY"
+	astUnsigned        = "UNSIGNED"
 )
 
 func (node *DDL) String() string {
 	switch node.Action {
-	case astCreateIndex, astDropIndex:
+	case astDropIndex:
 		return fmt.Sprintf("%s %s ON %s", node.Action, node.Name, node.NewName)
 	case astRenameTable:
 		return fmt.Sprintf("%s %s %s", node.Action, node.Name, node.NewName)
-	case astCreateDatabase, astCreateTable, astCreateView:
-		return fmt.Sprintf("%s %s", node.Action, node.NewName)
+	case astCreateView:
+		return fmt.Sprintf("%s%s %s", node.Action, node.IfNotExists, node.NewName)
+	case astShowDatabases:
```

Comment:
The vitess sql parser parses the MySQL variant of SQL. `SHOW DATABASES` comes from https://dev.mysql.com/doc/refman/5.1/en/show-databases.html. Namespace is not a term that is present in SQL. I'm not settled on this terminology, but it seems good to stick with the existing language for now.